### ISSUE TITLE
How to remove car from integration?

### ISSUE THREAD

**Issue by svalavuo** [Hyundai-Kia-Connect/kia_uvo#744]
Please check Services, Known Bug / Issues and Troubleshooting over here first: https://github.com/fuatakgun/kia_uvo/blob/master/README.md
**Region and Brand of car**
EU (Finland), Kia Niro
**Version of the integration**
2.14.1

**Describe the bug**
We bought a new Niro PHEV, but Integration shows the old Niro Hybrid as well.
How can I remove the old Niro from integration?
Old car doesn't show in Kia app.

**Debug logs if an error occurred**
No Log, just additiona car in devices.

Outlined here: https://github.com/fuatakgun/kia_uvo/blob/master/README.md

**To Reproduce**
Buy a new car and add it to your account.

**Expected behavior**
Old and removed car should disappear or at least user should be able to remove it from integration.

**Screenshots**

**Additional context**

**Comment by fuatakgun** [comment]
Let me work on this.

The old car is unavailable right?

**Comment by svalavuo** [comment]
Yes, that's correct. Car is unavailable in HA. Mobile app doesn't show it anymore.

Great work and nice to have access to car information and controls.
